elephant/asset/asset.py

```diff
@@ -1964,6 +1964,12 @@ class ASSET(object):
         If None, the attribute `t_stop` of the spike trains is used
         (if the same for all spike trains).
         Default: None
+    bin_tolerance : float or 'default' or None, optional
+        Defines the tolerance value for rounding errors when binning the
+        spike trains. If 'default', the value is the default as defined in
+        :class:`conv.BinnedSpikeTrain`. If None, no correction for binning
+        errors is performed. If a number, the binning will consider this value.
```

Comment:
```suggestion
        :class:`~.conversion.BinnedSpikeTrain`. If None, no correction for
        binning errors is performed. If a number, the binning will consider
        this value.
```

Get working link to `BinnedSpikeTrain` on readthedocs.

Reply:
Fixed in commit [8a090e8](https://github.com/NeuralEnsemble/elephant/pull/585/commits/8a090e88466b547206ae495e10c6e6f51dbe0dfe)